docs/reference.rst

```diff
@@ -116,3 +116,7 @@ Optional options:
 This section allows to set a custom list of key-value pairs that will be used
 as config data target attribute for device registration.
 They can be used for target filtering.
+
+  .. important::
```

Comment:
leading whitespace should not be required here as this directive block is at root level, isn't it?

Reply:
Probably? I don't know. So the "important note" for `post_update_reboot` is wrong, too? That's where I copied from :)

Reply:
This one is indented intentionally.

Reply:
Ah right, because the ``post_update_reboot=<boolean>`` functions as some kind of logical section there and the whole paragraph is indented.

Reply:
couldn't have said it better, yes ;)

Reply:
Fixed!